src/interactive/repl.ts

```diff
@@ -65,10 +65,10 @@ async function startREPL(preserveFocus: boolean, showTerminal: boolean = true) {
             return jlarg2
         }
 
-        const env = {
+        const env = Object.assign(process.env, {
```

Comment:
I think this one is good, we probably should have done that a long time ago :)

Reply:
Yeah, I was wondering about that too. This should be a no-op, except if VSCode messes with `process.env` -- [the docs](https://code.visualstudio.com/api/references/vscode-api#TerminalOptions) say that the environment should be inherited automatically
>env?: Object with environment variables that will be added to the VS Code process.
> strictEnv?: boolean
> Whether the terminal process environment should be exactly as provided in TerminalOptions.env. When this is false (default), the environment will be based on the window's environment and also apply configured platform settings like terminal.integrated.windows.env on top. When this is true, the complete environment must be provided as nothing will be inherited from the process or any configuration.

And it is, in most cases -- on cursory inspection the integrated REPL shows a normal environment. We did get reports about weird subtle differences between a shell env and the env in the REPL though, which I've always chalked up to us not starting Julia through a shell.

That said, doing this shouldn't hurt anything.